django_netjsonconfig/static/django-netjsonconfig/js/widget.js

```diff
@@ -93,13 +93,20 @@
     var isValidJson = function(advanced){
         var valid,
             context = {},
+            value,
             data,
             cleanedData;
         try {
             try {
-                context = JSON.parse($('#id_default_values').val());
+                value = $('#id_default_values').val();
+                if(value){
+                    context = JSON.parse(value);
```

Comment:
the indentation of this whole nested try/catch block is very messy and I don't understand what is going on: I don't understand if all the blocks have the closing `}` or not, please keep the indentation clean and the code readable